### PR TITLE
Fix node update conflict on startup

### PR DESCRIPTION
**Pull request by song-jiang**
## Description

We've seen node update conflict because startup.go was running the operation of setting node condition NetworkUnAvailable to false before updating node resource. This PR fixed that by moving the operation after node resources has been updated.

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Review by fasaxc — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Review by caseydavenport — APPROVED** [review]

**Pull request merged**
